src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBagInterface.php

```diff
@@ -82,6 +89,17 @@ public function set($name, $value);
     public function has($name);
 
     /**
+     * Removes a parameter.
+     *
+     * @param string $name The parameter name
+     *
+     * @throws LogicException if the parameter can not be removed
+     *
+     * @api
+     */
+    public function remove($name);
```

Comment:
This must be reverted. Adding a method to an interface is a BC break (classes implementing the interface but not the new method would raise an error).